src/lib/shared/input.ts

```diff
@@ -270,6 +270,24 @@ export interface InputPhrases {
 
   /** English: "Add value" */
   "input.valueAddButton": SimplePhrase;
+
+  /** English: "The value entered in this field will no longer be viewable once you click connect or save this page. However, you will still be able to enter a new value at any time." */
+  "input.writeOnly.alert": SimplePhrase;
+
+  /** English: "Replace Existing Value" */
```

Comment:
```suggestion
  /** English: "Replace existing value" */
```
In the designer and dashboard redesign we decided to always lower case proceeding words, however this sort have fallen by the waist side. We might want to come back to this later on and figure out what we want to do throughout the application.

Reply:
We might want to talk to UX/Product then about that, because a lot of the recent features that we have done have used a lot of uppercase

Reply:
Updated